### ISSUE TITLE
Scan-Hardware agent does not reflect changes in RAM amount in the DB

### ISSUE THREAD

**Issue by Deezzir** [ClusterLabs/anvil#262]
### Describe the bug
The `scan-hardware` agent is unable to update DB when the amount of RAM changes

### Steps to Reproduce
- Update the amount of available RAM on a system(strikers/nodes)
- The agent detects the changes and the entry appears in the logs

### Expected Behavior 
The agent should update the amount of available RAM in the DB and stop reporting the changes.

### Actual Behavior 
The following logs continue to constantly appear after RAM was updated on the system. It seems like the `scan-hardware` agent is unable to update the DB entry.
```
2022/12/13 19:41:39:[scan-hardware]:scan-hardware:818; The amount of RAM (as reported by dmidecode) on the system has increased. Likely the system was upgraded.
- New: ...... [2.05 GiB (2,198,863,872 bytes)]
- Old: ...... [2.00 GiB (2,147,483,648 bytes)]
- Difference: [49.00 MiB (51,380,224 bytes)]
2022/12/13 19:41:58:anvil-daemon:821; state_name: [db_in_use::9ea29df6-189d-41da-9ef4-4919b2f58473::3913650::/usr/sbin/scancore-agents/scan-hardware/scan-hardware]
2022/12/13 19:41:58:anvil-daemon:822; [ Warning ] - A no-longer active PID: [3913650] (used by: [/usr/sbin/scancore-agents/scan-hardware/scan-hardware] had marked the database: [an-striker01.alteeve.com (9ea29df6-189d-41da-9ef4-4919b2f58473)] as "in_use", but the PID is gone now. Reaping the flag.
2022/12/13 19:41:58:anvil-daemon:821; state_name: [db_in_use::c87012df-9b24-4d2d-8b72-1a10f68e9487::3913650::/usr/sbin/scancore-agents/scan-hardware/scan-hardware]
2022/12/13 19:41:58:anvil-daemon:822; [ Warning ] - A no-longer active PID: [3913650] (used by: [/usr/sbin/scancore-agents/scan-hardware/scan-hardware] had marked the database: [an-striker02.alteeve.com (c87012df-9b24-4d2d-8b72-1a10f68e9487)] as "in_use", but the PID is gone now. Reaping the flag.
```

**Comment by digimer** [comment]
Thanks, will fix asap

**Comment by digimer** [comment]
Resolved, please re-open if reappears after PR#320 is merged.
